package.json

```diff
@@ -114,9 +113,8 @@
   },
   "peerDependencies": {
     "moment": "^2.18.1",
-    "react": ">=0.14",
-    "react-dom": ">=0.14",
-    "react-addons-shallow-compare": ">=0.14"
+    "react": ">=15.3.0",
+    "react-dom": ">=15.3.0"
```

Comment:
What do you think is the estimated time to live on that? React 15 was currently released over a year ago (April 2016).